shared/middleware/net_sim_api.rb

```diff
@@ -79,7 +66,6 @@ def has_json_utf8_headers(request)
     table = get_table(shard_id, table_name)
     int_id = id.to_i
     table.delete(int_id)
```

Comment:
I'll see what I can do today.  Fortunately, I expect very few cascading deletes during our test today (one wire per deleted node, and maybe 0-2 messages).  You would see a lot more when deleting routers, or in limited-rate routing levels.